code/modules/urist/mob/spoopymobs.dm

```diff
@@ -21,11 +21,131 @@
 	move_to_delay = 7
 	stat_attack = 1
 	ranged = 0
-	universal_speak = 1
 	var/plague = 0 //whether biting dead people spawns new zombies
 	var/regen = 0 //if true, they won't stay down, but revive after a delay
 	var/regen_delay = 900 //delay for regen revive
 
+/datum/reagent/toxin/zombie/uristzombie
+	flags = AFFECTS_DEAD
+	metabolism = REM
+	target_organ = BP_BRAIN
+
+	var/symptom_msgs = list(
+		"Your teeth feel loose.",
+		"Your eyes are rheumy.",
+		"Your blood feels scratchy.",
+		"You joints feel stiff.",
+		"Bumps stand out on your skin.",
+		"Your intestines roil.",
+		"You feel light-headed.",
+		"Your throat hurts.",
+		"You feel fluid sloshing inside you.",
+		"Your chest feels tight.",
+		"You hear a crackling noise from inside of you.",
+		"Pus seeps from your eyes.",
+		"Your skin comes out in lesions.",
+		"The flesh at the back of your throat sloughs off.",
+		"Your joints ache.",
+		"Your fingernail falls off, the skin underneath blistered.",
+		"You taste iron."
+	)
+	var/transformation_msgs = list(
+		"Steel wool is scrubbing against your brain.",
+		"Greasy sand seems to fill your blood vessels.",
+		"Grey oil oozes out of your ears.",
+		"You smell spoiling meat. You drool at the thought.",
+		"Your tongue swells up and turns black."
+	)
+
+/datum/reagent/toxin/zombie/uristzombie/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
+	affect_blood(M, alien, removed * 0.5)
+
+/datum/reagent/toxin/zombie/uristzombie/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
```

Comment:
There's no way to delay or counteract this, dialysis isn't always avaliable - add dylovene as a delaying agent or peridaxon as a antidote?

Reply:
It's not supposed to be fair, it's supposed to be _fucking terrifying_. Rabies was a major inspiration - once you show symptoms, its recovery rate is single-digit, with brain damage, after damn near replacing the person's blood with antivirals, and we don't even know those weren't strokes of luck.

Peridaxon and Alkysine do act as delaying agents, and Dylovene helps the patient just not plain die on you regardless of transformation. Failing that, you can apply 10ccs of circular saw to the head - this kind of decision is a genre staple and I was explicitly designing around encouraging a scenario where you have to make it.

Reply:
As this is a subtype of reagent/toxin I'm almost certain dylovene is already an antidote for it. I'll check and see how effective this really is later.

Reply:
I checked the code, Dylovene removes all /reagent/toxin subtypes at a linear ratio of 1:4. On the other hand, the zombie chem exhibits pseudo-exponential growth (it's actually ridiculously complicated to find the expected value mathematically - it's a stack of two autoregressive models).

What this all amounts to is that a detox has a chance to work, especially if applied early and/or on a single bite or two. It still may work otherwise, but then it's basically Spess Milwaukee Protocol for Spess Zombie Rabies. And that balance makes me very happy.

Reply:
Turns out Dylovene had some special toxin-subtype reagent-clearing logic attached. It's no longer a subtype of that; I used Xenomicrobes since it makes some sense and has a chance of being blacklisted properly that way.